modules/nextflow/src/main/groovy/nextflow/script/FunctionDef.groovy

```diff
@@ -47,7 +51,7 @@ class FunctionDef extends ComponentDef implements ChainableDef {
 
     String getName() { alias }
 
-    BaseScript getOwner() { owner }
+    Object getOwner() { owner }
```

Comment:
I'm wrong, or nothing is using `getOwner()`. In that case I think it's better to remove it